back-end/storage/FirebaseDataParser.js

```diff
@@ -1,6 +1,6 @@
 class FirebaseDataParser {
     convertObjectArraysToArrays(data) {
-        if (typeof data !== 'object') {
+        if (data === null || typeof data !== 'object') {
```

Comment:
`typeof null === 'object'`?

Reply:
It is :scream:

Reply:
![](https://media.giphy.com/media/l9FAwJOXYprFe/giphy.gif)

Reply:
:D